### ISSUE TITLE
Cannot install plugin 

### ISSUE THREAD

**Issue by hsarji** [dgreif/homebridge-hatch-baby-rest#62]
I'm using an old Mac Mini as a server for Homebridge. The OS is Yosemite. It's working fine with Homebridge and other plugins, but I keep getting an error with the Hatch Baby Rest plugin. Here is an excerpt from the error log: 

```
2301 error errno 1
2302 error @abandonware/noble@1.9.2-14 install: `node-gyp rebuild`
2302 error Exit status 1
2303 error Failed at the @abandonware/noble@1.9.2-14 install script.
2303 error This is probably not a problem with npm. There is likely additional logging output above.
2304 verbose exit [ 1, true ]
```

I tried a manual install with `npm i -g homebridge-hatch-baby-rest` in terminal and am running into the same problems.

**Comment by dgreif** [comment]
Please see [the noble installation instructions](https://www.npmjs.com/package/@abandonware/noble#installation) and follow the steps for your platform.  Let me know if you still run into issues after that

**Comment by hsarji** [comment]
I was able to install the plugin. Noble requires Xcode. I downloaded Xcode and reinstalled the plugin. But the plugin isn't working. I see it in the Home app, but turning it on or adjusting the volume does nothing. This seems to be a duplicate of [another open issue](https://github.com/dgreif/homebridge-hatch-baby-rest/issues/59), so I will close this issue with this comment. Thank you for helping.

**Comment by dgreif** [comment]
@hsarji I don't think this is the same as #59.  Could you post more logs now that you have it installed?
